test/unit/core.js

```diff
@@ -197,6 +197,34 @@ QUnit.test( "globalEval execution after script injection (#7862)", function( ass
 	assert.ok( window.strictEvalTest - now < 500, "Code executed synchronously" );
 } );
 
+QUnit.test( "jQuery.append with crossOrigin attribute", function( assert ) {
+	var done = assert.async();
+	assert.expect( 1 );
+
+	var timeout = null;
+
+	Globals.register( "corsCallback" );
+	window.corsCallback = function( response ) {
+		assert.ok( response === true, "Cors" );
+
+		if ( timeout != null ) {
+			window.clearTimeout( timeout );
+		}
+
+		done();
+	};
+
+	var src = baseURL + "mock.php?action=cors";
+	src = src.replace( "localhost", "127.0.0.1" );
+	var html = "<script type=\"text/javascript\" src=\"" + src + "\" crossOrigin=\"anonymous\"><\/script>";
+
+	jQuery( document.body ).append( html );
+	timeout = window.setTimeout( function() {
+		assert.ok( false, "Cors" );
```

Comment:
This assertion fires for me in Safari 13, Edge 18 & IE 11. It correctly doesn't fire only in Firefox 72 & Chrome 79.

We need to figure out why it fails in other browsers. For IE we might have to skip the test as it doesn't support the crossorigin attribute (by switching from `QUnit.test` to `QUnit.testUnlessIE`). Other browsers should pass it, though.